src/komponenter/modal/AvbrytAvtaleModal.tsx

```diff
@@ -13,11 +13,11 @@ import React, { FunctionComponent, useEffect, useState } from 'react';
 import './AvbrytAvtaleModal.less';
 import BekreftelseModal from './BekreftelseModal';
 
-type Props = {
+interface Props {
     isOpen: boolean;
     lukkModal: () => void;
     avbrytAvtale: (avbruttDato: string, avbruttGrunn: string) => Promise<any>;
-};
+}
```

Comment:
Hvorfor er denne byttet fra type til interface?
Virker som det er ganske utbredt enighet om at type er godt nok nå, spesielt for function components.